### ISSUE TITLE
Issue if folder has Slovenian character

### ISSUE THREAD

**Issue by kacnje** [TechieGuy12/FileWatcher#24]
If in <watch> section in XML I use <path>k:\zacasno</path>, it works. If I change it to <path>k:\začasno</path>, I get an error: _The configuration file could not be read. Reason: There is an error in XML document (3, 20)._

Is this a bug?

**Comment by TechieGuy12** [comment]
It is an issue with character with the accent. I'll have to test on my side to determine the fix.

**Comment by kacnje** [comment]
Tnx. We will wait to see results.

**Comment by kacnje** [comment]
Of couse:```

<watches>
	<watch>
	        <path>K:\Začasno</path>
        <commands>
            <command>
                <triggers>
			<trigger>Create</trigger>
			<trigger>Change</trigger>
			<trigger>Rename</trigger>
			<trigger>Delete</trigger>
                </triggers>            
		<path>C:\Program Files\Amebis Besana\BesPre.exe</path>
            </command>
        </commands>    
    </watch>
</watches>
```

If I change the watch folder to something without Slovenian character, it works.

**Comment by TechieGuy12** [comment]
I tried the XML, and still I can't reproduce the error. What encoding are you using for the XML file - UTF-8?

**Comment by kacnje** [comment]
Text encoding was "windows-1250". I saved the XML file into "UTF-8" encoding and it worked. 
After celebrating moments, I changed the line:
`<path>C:\Program Files\Amebis Besana\BesPre.exe</path>`
to 
`<path>msg /server:computer1 user1 Notification</path>`

And now I get:
`ERROR The command 'msg /server:comp1 john Notification' was not found. Command was not run.`
Note: _comp1_ and _john_ are changed here to protect privacy. 

If I run the same command in CMD prompt, it works. Could be something wrong with the slash character when using UTF-8 encoding. It worked before.
